.gitignore

```diff
@@ -84,3 +84,8 @@ venv.bak/
 
 # In case you place source code in ~/nni/
 /experiments
+src/sdk/pynni/nni/nni
```

Comment:
I think these are wrong soft links. We should investigate in why they are created.